environment/stats/src/cloud/benchflow/collectors/stats.go

```diff
@@ -176,10 +181,10 @@ func startCollecting(w http.ResponseWriter, r *http.Request) {
 	for _, each := range conts {
 		containerInspect, err := client.InspectContainer(each)
 		networks := containerInspect.NetworkSettings.Networks
-		isHost := false
+		network := "bridge"
```

Comment:
As per in https://docs.docker.com/engine/reference/commandline/run/, the possible values are:

```
--net="bridge"                Connect a container to a network
                                'bridge': create a network stack on the default Docker bridge
                                'none': no networking
                                'container:<name|id>': reuse another container's network stack
                                'host': use the Docker host network stack
                                '<network-name>|<network-id>': connect to a user-defined network
```

For now just document it with a comment in the code, it is fine to assume `bridge` as default as it is the one used by Docker, and then switch to `host` in case it is.